e2e/test/scenarios/dashboard-cards/click-behavior.cy.spec.js

```diff
@@ -608,6 +672,646 @@ describe("scenarios > dashboard > dashboard cards > click behavior", () => {
       });
     });
   });
+
+  describe("table", () => {
+    const questionDetails = QUESTION_TABLE;
+    const dashboardDetails = {
+      parameters: [DASHBOARD_FILTER_TEXT],
+    };
+
+    it("should open drill-through menu as a default click-behavior", () => {
+      cy.createQuestionAndDashboard({ questionDetails }).then(
+        ({ body: card }) => {
+          visitDashboard(card.dashboard_id);
+        },
+      );
+
+      getTableCell(COLUMN_INDEX.COUNT).click();
+      popover().should("contain.text", "Filter by this value");
+
+      getTableCell(COLUMN_INDEX.CREATED_AT).click();
+      popover().should("contain.text", "Filter by this date");
+
+      editDashboard();
+
+      getDashboardCard().realHover().icon("click").click();
+      getDashboardCard()
+        .button()
+        .should("have.text", "Open the drill-through menu");
+    });
+
+    it("should allow setting dashboard and saved question as custom destination for different columns", () => {
```

Comment:
I wanted to test more columns in one `it()` but the code became completely unmanageable, so I limited it to testing 2 columns at a time.